common/src/main/kotlin/tools/samt/common/Location.kt

```diff
@@ -1,15 +1,21 @@
 package tools.samt.common
 
 data class FileOffset(val charIndex: Int, val row: Int, val col: Int) {
-    override fun toString(): String = "$row:$col"
+    override fun toString(): String = "${row + 1}:${col + 1}"
 }
 
 data class Location(
-    val start: FileOffset,
-    val end: FileOffset = start,
+    var start: FileOffset,
+    var end: FileOffset = start,
 ) {
     init {
-        require(start.row <= end.row) { "Location end row (${end.row}) cannot be before start row ${start.row}" }
+        if (start.row > end.row) {
+            end = start
+        }
+
+        // FIXME: There is some bug with this enabled
```

Comment:
I would like to fix this before merging this, just making this comment as a reminder